wolfssh/port.h

```diff
@@ -1182,7 +1182,11 @@ extern "C" {
 
 #elif defined(WOLFSSH_USER_FILESYSTEM)
     /* User-defined I/O support */
-    #include "myFilesystem.h"
+    #ifdef WOLFSSH_STM32_CUBEMX
+        #include <wolfssh/myFilesystem.h>
```

Comment:
Could this instead be fixed with an include path adjustment? Did you actually get the filesystem working with the wolfSSH Cube pack?

Reply:
Just building. For the default filesystem stuff, I took the same approach as https://github.com/wolfSSL/wolfssh/tree/master/ide/IAR-EWARM with a dummy filesystem and leave it up to the user to set it up (`WOLFSSH_USER_FILESYSTEM`). The build works with `WOLFSSH_FATFS` as well. As far as running the filesystem tests, they are dependent on `wolfssl/test.h` which isn't in the wolfSSL cube pack. I'm sure there's a way around that but haven't dived deep into it